app/api/events.py

```diff
@@ -210,15 +278,33 @@ def before_get_object(self, view_kwargs):
                 else:
                     view_kwargs['id'] = None
 
-    decorators = (is_coorganizer, )
+    def after_get(self, result):
+        result = changeToTimezone(result)
```

Comment:
@niranjan94 

If the data is : 

```
"starts-at":"2017-06-12T12:00:00+06:30",
"ends-at":"2017-06-12T12:00:00+06:00",
"timezone":"Asia/Kolkata"
```

In database it is like ---

```
starts_at : 2017-06-12 11:00:00+05:30
ends_at : 2017-06-12 11:30:00+05:30
timezone : Asia/Kolkata
```

That is because my localtime ( server timezone ) is Asia/Kolkata.

And if I change my localtime to BST ( British Summer Time ) it changes accordingly like this 
```
starts_at : 2017-06-12 06:30:00+01
ends_at : 2017-06-12 06:30:00+01
timezone : Asia/Kolkata
```

Did you meant something different @niranjan94 ?